### DISCUSSION TITLE
New flow for translatable fields

### DISCUSSION THREAD

**Discussion by federicocappellotto97** [directus/directus#9829] [Feature Requests]
Let's start with an example, my frontend has **title**, **image**, **editor**, in this order.
I like to put fields on backend in the same order, but in this case only **title** and **editor** are translatable field. With current flow i need to put also the image field as translatable (or i need to update to the bottom, breaking the correct fields order).

The best thing would be mix translatable and normal fields together.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
